src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -181,3 +146,83 @@ export const thorchainSaversStakingOpportunitiesMetadataResolver = async ({
 
   return { data }
 }
+
+export const thorchainSaversStakingOpportunitiesUserDataResolver = async ({
+  opportunityType,
```

Comment:
do we need to inject this? we know it's just staking?

Reply:
We need to actually pass it to make `getOpportunitiesUserDataResolversByDefiProviderAndDefiType` happy, since it will pass arguments down, but yeah in theory we could just hardcode it